healthcheck/healthcheck_test.go

```diff
@@ -182,12 +111,12 @@ func TestSetBackendsConfiguration(t *testing.T) {
 
 			lb.Lock()
 			defer lb.Unlock()
-			if lb.numRemovedServers != test.wantNumRemovedServers {
-				t.Errorf("got %d removed servers, wanted %d", lb.numRemovedServers, test.wantNumRemovedServers)
+			if test.wantNumRemovedServers != lb.numRemovedServers {
```

Comment:
The order in Go is [_got first, want second_](https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures); both in the comparison expression and the error message.